account_payment_restrict_mandate/tests/test_account_payment_restrict_mandate.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
+
+
+class TestAccountPaymentRestrictMandate(TransactionCase):
+    def test_account_payment_restrict_mandate(self):
+        mandate = self.env['account.banking.mandate'].create({
+            'partner_bank_id': self.env.ref('base.res_partner_2').bank_ids
+            .filtered(lambda x: x.state == 'iban').id,
+            'max_amount_per_date': 600,
+            'date_start': '2016-01-01 00:00:00',
```

Comment:
I was thinking about the missing fiscal year, but now it occurred to me
that this probably is not a problem because no moves are actually
created in the test.